toolbox/gpu-operator/operand-build/operands/device_plugin.json

```diff
@@ -0,0 +1,19 @@
+{
+  "build_operand_git_repo": "https://gitlab.com/nvidia/kubernetes/device-plugin.git",
+  "build_operand_git_ref": "master",
+  "build_operand_namespace": "gpu-operator-resources",
+  "build_operand_imagestream_name": "k8s-device-plugin",
+  "build_operand_tag": "latest",
+  "build_operand_dockerfile": "docker/Dockerfile.ubi8",
+  "build_operand_docker_context": ".",
+  "build_operand_docker_build_args": [
+    {
+      "name": "GOLANG_VERSION",
+      "value": "1.15.8"
```

Comment:
An example of a "bad" hardcoded value we might want to retrieve in a smarter way, as mentioned in [this comment](https://github.com/openshift-psap/ci-artifacts/pull/172/files#r637164014)